scripts/build/templates/icons-json.d.ts.template

```diff
@@ -0,0 +1,6 @@
+import type {IconData} from '../types.js';
```

Comment:
I think that this doesn't require a `.template` file because is not a true template as the file is rendered as is. Several options here:

1. Remove `.template` suffix from the file name so we can have proper syntax highlighting, lint, etc.
2. Put it into the *data/* folder as a regular TS file. The problem with that approach is that it could confuse contributors, as is not a data file.
3. Put it into the root of the project as a regular TS file.
4. Render it from a string directly from the build script.

My preference is, in this order: 3, 2, 1 and 4.

Reply:
Any reason why this was resolved, @LitoMore?